Sons of Eridanus update

Delnth opened the pull request:
Add an SOE ship, change URFC to SOE, add spawnpoints, delete the URFC Avarice.

Also biogenerators can now print advanced burn/trauma kits for 100 units of bio matter.

Delnth committed:
Sons of Eridanus update

Add an SOE ship, change URFC to SOE, add spawnpoints, delete the URFC Avarice.

Also biogenerators can now print advanced burn/trauma kits for 100 units of bio matter.

Delnth committed:
Changelog

title

caelaislinn reviewed (changes requested):
I'm not sure if that spawn will work, did you test it? Also I need you to delete the backup map files generated by the avarice-osprey mapping changes. And I'd prefer you not delete entire map files for now, just leave them unused.

Delnth commented:
I have tested everything. Haven't noted any problem so far, except the ship guns that refuse to work despite XO's help.

Deleting the backup files.

Delnth committed:
Delete the Avarice backup files

title

Delnth committed:
Mapping/job QoL

Add new job ranks/titles for SOE and tweak the ship map

Delnth committed:
More map QoL

Fix the empty chem dispenser and add portable air pumps as well as air pressure tank to supply them; in order to restore a safe environment once a breach is patched.

caelaislinn reviewed (changes requested):
Can you also please restore second_deck.dmm ? We can leave it unused and just load the first deck

Delnth committed:
re-add second_deck.dmm of the avarice as per request

Delnth committed:
re-add first_deck.dmm of the Avarice as per request (should stop deleting its own content now)
